src/orquestra/sdk/_base/_dsl.py

```diff
@@ -109,9 +122,19 @@ def infer(
         return DeferredGitImport(local_repo_path, git_ref)
 
 
-def GithubImport(repo: str, git_ref: str = "main"):
+def GithubImport(
+    repo: str,
+    git_ref: str = "main",
+    username: Optional[str] = None,
+    personal_access_token: Optional[Secret] = None,
```

Comment:
I was thinking about adding a defensive check on this:
```
if not isinstance(personal_access_token, Secret):
    raise TypeError(f"A Github personal access token should be passed as an `sdk.Secret`, got {type(personal_access_token}")
```
but this would cause exceptions during imports because `GithubImport` is usually put inside decorators.

Type checkers should show this and later the graph traversal code will fail if anything but a `sdk.Secret` was passed.